.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: python
+python:
+  - 3.6
+  - 3.7-dev
+
+notifications:
+  email: false
+
+#setuptools are broken for 3.7
+jobs:
+  allow_failures:
+    - python: 3.7-dev
```

Comment:
Seems counter-productive, are we able to remove Python 3.7 from testing until this is resolved?

Reply:
True, and add back when issue with setuptools is resolved (for which they say soon)

Reply:
soon:tm:

Reply:
yup, *sad*